hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -236,6 +236,21 @@ public static void initClass() throws Exception {
     prepareParquetDFSFiles(PARQUET_NUM_RECORDS, PARQUET_SOURCE_ROOT);
   }
 
+  @AfterAll
```

Comment:
Can we avoid changes in this class since there are no actual changes ?

Reply:
sure